Move SocketStreamHandle and WebSocketChannel to WebKitLegacy

achristensen07 opened the pull request:
#### b1f1fbf2ed7c2415e59354cc49d7e76cff5d9d67
<pre>
Move SocketStreamHandle and WebSocketChannel to WebKitLegacy
<a href="https://bugs.webkit.org/show_bug.cgi?id=254221">https://bugs.webkit.org/show_bug.cgi?id=254221</a>
rdar://107003946

Reviewed by Youenn Fablet.

* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/websockets/ThreadableWebSocketChannel.cpp:
* Source/WebCore/Modules/websockets/ThreadableWebSocketChannel.h:
* Source/WebCore/Modules/websockets/WebSocket.cpp:
(WebCore::WebSocket::close):
(WebCore::WebSocket::didConnect):
(WebCore::WebSocket::didClose):
* Source/WebCore/Modules/websockets/WebSocketHandshake.h:
* Source/WebCore/Modules/websockets/WorkerThreadableWebSocketChannel.cpp:
* Source/WebCore/PlatformMac.cmake:
* Source/WebCore/Sources.txt:
* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/dom/Document.h:
* Source/WebCore/inspector/LegacyWebSocketInspectorInstrumentation.cpp: Added.
(WebCore::LegacyWebSocketInspectorInstrumentation::hasFrontends):
(WebCore::LegacyWebSocketInspectorInstrumentation::didCreateWebSocket):
(WebCore::LegacyWebSocketInspectorInstrumentation::willSendWebSocketHandshakeRequest):
(WebCore::LegacyWebSocketInspectorInstrumentation::didReceiveWebSocketHandshakeResponse):
(WebCore::LegacyWebSocketInspectorInstrumentation::didCloseWebSocket):
(WebCore::LegacyWebSocketInspectorInstrumentation::didReceiveWebSocketFrame):
(WebCore::LegacyWebSocketInspectorInstrumentation::didSendWebSocketFrame):
(WebCore::LegacyWebSocketInspectorInstrumentation::didReceiveWebSocketFrameError):
* Source/WebCore/inspector/LegacyWebSocketInspectorInstrumentation.h: Added.
* Source/WebCore/inspector/agents/InspectorNetworkAgent.cpp:
* Source/WebCore/inspector/agents/page/PageNetworkAgent.cpp:
* Source/WebCore/page/SocketProvider.cpp:
(WebCore::SocketProvider::createWebSocketChannel):
* Source/WebCore/page/SocketProvider.h:
* Source/WebCore/platform/Curl.cmake:
* Source/WebCore/platform/Soup.cmake:
* Source/WebCore/platform/network/curl/SocketStreamHandleImpl.h: Removed.
* Source/WebCore/platform/network/soup/SocketStreamHandleImpl.h: Removed.
* Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm:
* Source/WebKit/NetworkProcess/cocoa/WebSocketTaskCocoa.mm:
(WebKit::WebSocketTask::readNextMessage):
(WebKit::WebSocketTask::close):
* Source/WebKit/WebProcess/Network/WebSocketChannel.cpp:
(WebKit::WebSocketChannel::close):
(WebKit::WebSocketChannel::fail):
(WebKit::WebSocketChannel::disconnect):
(WebKit::WebSocketChannel::didClose):
* Source/WebKitLegacy/Sources.txt:
* Source/WebKitLegacy/WebCoreSupport/LegacySocketProvider.cpp: Copied from Source/WebCore/page/SocketProvider.cpp.
(LegacySocketProvider::createWebSocketChannel):
* Source/WebKitLegacy/WebCoreSupport/LegacySocketProvider.h: Copied from Source/WebCore/page/SocketProvider.cpp.
(LegacySocketProvider::create):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandle.cpp: Renamed from Source/WebCore/platform/network/SocketStreamHandle.cpp.
(WebCore::SocketStreamHandle::SocketStreamHandle):
(WebCore::SocketStreamHandle::state const):
(WebCore::SocketStreamHandle::sendData):
(WebCore::SocketStreamHandle::sendHandshake):
(WebCore::SocketStreamHandle::close):
(WebCore::SocketStreamHandle::disconnect):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandle.h: Renamed from Source/WebCore/platform/network/SocketStreamHandle.h.
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleClient.h: Renamed from Source/WebCore/platform/network/SocketStreamHandleClient.h.
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.cpp: Renamed from Source/WebCore/platform/network/SocketStreamHandleImpl.cpp.
(WebCore::SocketStreamHandleImpl::platformSend):
(WebCore::removeTerminationCharacters):
(WebCore::cookieDataForHandshake):
(WebCore::SocketStreamHandleImpl::platformSendHandshake):
(WebCore::SocketStreamHandleImpl::sendPendingData):
(WebCore::SocketStreamHandleImpl::bufferedAmount):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.h: Renamed from Source/WebCore/platform/network/cf/SocketStreamHandleImpl.h.
(WebCore::SocketStreamHandleImpl::create):
(WebCore::SocketStreamHandleImpl::shouldUseSSL const):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImplCFNet.cpp: Renamed from Source/WebCore/platform/network/cf/SocketStreamHandleImplCFNet.cpp.
(WebCore::callbacksRunLoop):
(WebCore::callbacksRunLoopMode):
(WebCore::SocketStreamHandleImpl::SocketStreamHandleImpl):
(WebCore::SocketStreamHandleImpl::scheduleStreams):
(WebCore::SocketStreamHandleImpl::retainSocketStreamHandle):
(WebCore::SocketStreamHandleImpl::releaseSocketStreamHandle):
(WebCore::SocketStreamHandleImpl::copyPACExecutionDescription):
(WebCore::MainThreadPACCallbackInfo::MainThreadPACCallbackInfo):
(WebCore::SocketStreamHandleImpl::pacExecutionCallback):
(WebCore::SocketStreamHandleImpl::executePACFileURL):
(WebCore::SocketStreamHandleImpl::removePACRunLoopSource):
(WebCore::SocketStreamHandleImpl::chooseProxy):
(WebCore::SocketStreamHandleImpl::chooseProxyFromArray):
(WebCore::setCONNECTProxyForStream):
(WebCore::SocketStreamHandleImpl::setLegacyTLSEnabled):
(WebCore::SocketStreamHandleImpl::createStreams):
(WebCore::SocketStreamHandleImpl::getStoredCONNECTProxyCredentials):
(WebCore::authenticationSchemeFromAuthenticationMethod):
(WebCore::setCONNECTProxyAuthorizationForStream):
(WebCore::SocketStreamHandleImpl::addCONNECTCredentials):
(WebCore::SocketStreamHandleImpl::copyCFStreamDescription):
(WebCore::SocketStreamHandleImpl::readStreamCallback):
(WebCore::SocketStreamHandleImpl::writeStreamCallback):
(WebCore::setResponseProxyURL):
(WebCore::copyCONNECTProxyResponse):
(WebCore::SocketStreamHandleImpl::reportErrorToClient):
(WebCore::SocketStreamHandleImpl::~SocketStreamHandleImpl):
(WebCore::SocketStreamHandleImpl::platformSendInternal):
(WebCore::SocketStreamHandleImpl::platformClose):
(WebCore::SocketStreamHandleImpl::port const):
* Source/WebKitLegacy/WebCoreSupport/WebSocketChannel.cpp: Renamed from Source/WebCore/Modules/websockets/WebSocketChannel.cpp.
(WebCore::WebSocketChannel::WebSocketChannel):
(WebCore::WebSocketChannel::~WebSocketChannel):
(WebCore::WebSocketChannel::connect):
(WebCore::WebSocketChannel::document):
(WebCore::WebSocketChannel::subprotocol):
(WebCore::WebSocketChannel::extensions):
(WebCore::WebSocketChannel::send):
(WebCore::WebSocketChannel::bufferedAmount const):
(WebCore::WebSocketChannel::close):
(WebCore::WebSocketChannel::fail):
(WebCore::WebSocketChannel::disconnect):
(WebCore::WebSocketChannel::suspend):
(WebCore::WebSocketChannel::resume):
(WebCore::WebSocketChannel::didOpenSocketStream):
(WebCore::WebSocketChannel::didCloseSocketStream):
(WebCore::WebSocketChannel::didReceiveSocketStreamData):
(WebCore::WebSocketChannel::didFailToReceiveSocketStreamData):
(WebCore::WebSocketChannel::didUpdateBufferedAmount):
(WebCore::WebSocketChannel::didFailSocketStream):
(WebCore::WebSocketChannel::didStartLoading):
(WebCore::WebSocketChannel::didReceiveData):
(WebCore::WebSocketChannel::didFinishLoading):
(WebCore::WebSocketChannel::didFail):
(WebCore::WebSocketChannel::appendToBuffer):
(WebCore::WebSocketChannel::skipBuffer):
(WebCore::WebSocketChannel::processBuffer):
(WebCore::WebSocketChannel::resumeTimerFired):
(WebCore::WebSocketChannel::startClosingHandshake):
(WebCore::WebSocketChannel::closingTimerFired):
(WebCore::WebSocketChannel::processFrame):
(WebCore::WebSocketChannel::enqueueTextFrame):
(WebCore::WebSocketChannel::enqueueRawFrame):
(WebCore::WebSocketChannel::enqueueBlobFrame):
(WebCore::WebSocketChannel::processOutgoingFrameQueue):
(WebCore::WebSocketChannel::abortOutgoingFrameQueue):
(WebCore::WebSocketChannel::sendFrame):
(WebCore::WebSocketChannel::clientHandshakeRequest const):
(WebCore::WebSocketChannel::serverHandshakeResponse const):
* Source/WebKitLegacy/WebCoreSupport/WebSocketChannel.h: Renamed from Source/WebCore/Modules/websockets/WebSocketChannel.h.
* Source/WebKitLegacy/WebKitLegacy.xcodeproj/project.pbxproj:
* Source/WebKitLegacy/mac/WebView/WebView.mm:
(-[WebView _commonInitializationWithFrameName:groupName:]):
(-[WebView initSimpleHTMLDocumentWithStyle:frame:preferences:groupName:]):

Canonical link: <a href="https://commits.webkit.org/261973@main">https://commits.webkit.org/261973@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e2d897ee3b62c2a42b83961f73dde8e849ad7aa3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/152 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/161 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/167 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/154 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/137 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/154 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/155 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/152 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/237 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/156 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/120 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/146 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/169 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/152 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/143 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/135 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/133 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/149 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/152 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/145 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/120 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/127 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/146 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/149 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/14 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

youennf commented on Source/WebCore/Modules/websockets/ThreadableWebSocketChannel.h:
```diff
@@ -93,6 +93,25 @@ class ThreadableWebSocketChannel {
     virtual ResourceRequest clientHandshakeRequest(const CookieGetter&) const = 0;
     virtual const ResourceResponse& serverHandshakeResponse() const = 0;
 
+    enum CloseEventCode {
```
Can we enum class it?

achristensen07 replied:
This is quite messy.  There is a lot of implicit conversion to/from int, unsigned short, short, etc.  I think it should be done separately.

youennf commented on Source/WebKitLegacy/WebCoreSupport/LegacySocketProvider.h:
```diff
@@ -0,0 +1,35 @@
+
+#include <WebCore/SocketProvider.h>
+
+class LegacySocketProvider : public WebCore::SocketProvider {
```
final

youennf commented on Source/WebKitLegacy/WebCoreSupport/LegacySocketProvider.h:
```diff
@@ -0,0 +1,35 @@
+public:
+    static Ref<LegacySocketProvider> create() { return adoptRef(*new LegacySocketProvider); }
+private:
+    RefPtr<WebCore::ThreadableWebSocketChannel> createWebSocketChannel(WebCore::Document&, WebCore::WebSocketChannelClient&);
```
final

youennf commented on Source/WebCore/page/SocketProvider.cpp:
```diff
@@ -26,16 +26,13 @@
 {
-    // FIXME: Move this to WebKitLegacy.
-    return WebCore::WebSocketChannel::create(document, client, *this);
+    return nullptr;
```
pure virtual and an EmptySocketProvider would probably be clearer.

achristensen07 committed:
Move SocketStreamHandle and WebSocketChannel to WebKitLegacy
https://bugs.webkit.org/show_bug.cgi?id=254221
rdar://107003946

Reviewed by Youenn Fablet.

* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/websockets/ThreadableWebSocketChannel.cpp:
* Source/WebCore/Modules/websockets/ThreadableWebSocketChannel.h:
* Source/WebCore/Modules/websockets/WebSocket.cpp:
(WebCore::WebSocket::close):
(WebCore::WebSocket::didConnect):
(WebCore::WebSocket::didClose):
* Source/WebCore/Modules/websockets/WebSocketHandshake.h:
* Source/WebCore/Modules/websockets/WorkerThreadableWebSocketChannel.cpp:
* Source/WebCore/PlatformMac.cmake:
* Source/WebCore/Sources.txt:
* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/dom/Document.h:
* Source/WebCore/inspector/LegacyWebSocketInspectorInstrumentation.cpp: Added.
(WebCore::LegacyWebSocketInspectorInstrumentation::hasFrontends):
(WebCore::LegacyWebSocketInspectorInstrumentation::didCreateWebSocket):
(WebCore::LegacyWebSocketInspectorInstrumentation::willSendWebSocketHandshakeRequest):
(WebCore::LegacyWebSocketInspectorInstrumentation::didReceiveWebSocketHandshakeResponse):
(WebCore::LegacyWebSocketInspectorInstrumentation::didCloseWebSocket):
(WebCore::LegacyWebSocketInspectorInstrumentation::didReceiveWebSocketFrame):
(WebCore::LegacyWebSocketInspectorInstrumentation::didSendWebSocketFrame):
(WebCore::LegacyWebSocketInspectorInstrumentation::didReceiveWebSocketFrameError):
* Source/WebCore/inspector/LegacyWebSocketInspectorInstrumentation.h: Added.
* Source/WebCore/inspector/agents/InspectorNetworkAgent.cpp:
* Source/WebCore/inspector/agents/page/PageNetworkAgent.cpp:
* Source/WebCore/page/SocketProvider.cpp:
(WebCore::SocketProvider::createWebSocketChannel):
* Source/WebCore/page/SocketProvider.h:
* Source/WebCore/platform/Curl.cmake:
* Source/WebCore/platform/Soup.cmake:
* Source/WebCore/platform/network/curl/SocketStreamHandleImpl.h: Removed.
* Source/WebCore/platform/network/soup/SocketStreamHandleImpl.h: Removed.
* Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm:
* Source/WebKit/NetworkProcess/cocoa/WebSocketTaskCocoa.mm:
(WebKit::WebSocketTask::readNextMessage):
(WebKit::WebSocketTask::close):
* Source/WebKit/WebProcess/Network/WebSocketChannel.cpp:
(WebKit::WebSocketChannel::close):
(WebKit::WebSocketChannel::fail):
(WebKit::WebSocketChannel::disconnect):
(WebKit::WebSocketChannel::didClose):
* Source/WebKitLegacy/Sources.txt:
* Source/WebKitLegacy/WebCoreSupport/LegacySocketProvider.cpp: Copied from Source/WebCore/page/SocketProvider.cpp.
(LegacySocketProvider::createWebSocketChannel):
* Source/WebKitLegacy/WebCoreSupport/LegacySocketProvider.h: Copied from Source/WebCore/page/SocketProvider.cpp.
(LegacySocketProvider::create):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandle.cpp: Renamed from Source/WebCore/platform/network/SocketStreamHandle.cpp.
(WebCore::SocketStreamHandle::SocketStreamHandle):
(WebCore::SocketStreamHandle::state const):
(WebCore::SocketStreamHandle::sendData):
(WebCore::SocketStreamHandle::sendHandshake):
(WebCore::SocketStreamHandle::close):
(WebCore::SocketStreamHandle::disconnect):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandle.h: Renamed from Source/WebCore/platform/network/SocketStreamHandle.h.
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleClient.h: Renamed from Source/WebCore/platform/network/SocketStreamHandleClient.h.
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.cpp: Renamed from Source/WebCore/platform/network/SocketStreamHandleImpl.cpp.
(WebCore::SocketStreamHandleImpl::platformSend):
(WebCore::removeTerminationCharacters):
(WebCore::cookieDataForHandshake):
(WebCore::SocketStreamHandleImpl::platformSendHandshake):
(WebCore::SocketStreamHandleImpl::sendPendingData):
(WebCore::SocketStreamHandleImpl::bufferedAmount):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.h: Renamed from Source/WebCore/platform/network/cf/SocketStreamHandleImpl.h.
(WebCore::SocketStreamHandleImpl::create):
(WebCore::SocketStreamHandleImpl::shouldUseSSL const):
* Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImplCFNet.cpp: Renamed from Source/WebCore/platform/network/cf/SocketStreamHandleImplCFNet.cpp.
(WebCore::callbacksRunLoop):
(WebCore::callbacksRunLoopMode):
(WebCore::SocketStreamHandleImpl::SocketStreamHandleImpl):
(WebCore::SocketStreamHandleImpl::scheduleStreams):
(WebCore::SocketStreamHandleImpl::retainSocketStreamHandle):
(WebCore::SocketStreamHandleImpl::releaseSocketStreamHandle):
(WebCore::SocketStreamHandleImpl::copyPACExecutionDescription):
(WebCore::MainThreadPACCallbackInfo::MainThreadPACCallbackInfo):
(WebCore::SocketStreamHandleImpl::pacExecutionCallback):
(WebCore::SocketStreamHandleImpl::executePACFileURL):
(WebCore::SocketStreamHandleImpl::removePACRunLoopSource):
(WebCore::SocketStreamHandleImpl::chooseProxy):
(WebCore::SocketStreamHandleImpl::chooseProxyFromArray):
(WebCore::setCONNECTProxyForStream):
(WebCore::SocketStreamHandleImpl::setLegacyTLSEnabled):
(WebCore::SocketStreamHandleImpl::createStreams):
(WebCore::SocketStreamHandleImpl::getStoredCONNECTProxyCredentials):
(WebCore::authenticationSchemeFromAuthenticationMethod):
(WebCore::setCONNECTProxyAuthorizationForStream):
(WebCore::SocketStreamHandleImpl::addCONNECTCredentials):
(WebCore::SocketStreamHandleImpl::copyCFStreamDescription):
(WebCore::SocketStreamHandleImpl::readStreamCallback):
(WebCore::SocketStreamHandleImpl::writeStreamCallback):
(WebCore::setResponseProxyURL):
(WebCore::copyCONNECTProxyResponse):
(WebCore::SocketStreamHandleImpl::reportErrorToClient):
(WebCore::SocketStreamHandleImpl::~SocketStreamHandleImpl):
(WebCore::SocketStreamHandleImpl::platformSendInternal):
(WebCore::SocketStreamHandleImpl::platformClose):
(WebCore::SocketStreamHandleImpl::port const):
* Source/WebKitLegacy/WebCoreSupport/WebSocketChannel.cpp: Renamed from Source/WebCore/Modules/websockets/WebSocketChannel.cpp.
(WebCore::WebSocketChannel::WebSocketChannel):
(WebCore::WebSocketChannel::~WebSocketChannel):
(WebCore::WebSocketChannel::connect):
(WebCore::WebSocketChannel::document):
(WebCore::WebSocketChannel::subprotocol):
(WebCore::WebSocketChannel::extensions):
(WebCore::WebSocketChannel::send):
(WebCore::WebSocketChannel::bufferedAmount const):
(WebCore::WebSocketChannel::close):
(WebCore::WebSocketChannel::fail):
(WebCore::WebSocketChannel::disconnect):
(WebCore::WebSocketChannel::suspend):
(WebCore::WebSocketChannel::resume):
(WebCore::WebSocketChannel::didOpenSocketStream):
(WebCore::WebSocketChannel::didCloseSocketStream):
(WebCore::WebSocketChannel::didReceiveSocketStreamData):
(WebCore::WebSocketChannel::didFailToReceiveSocketStreamData):
(WebCore::WebSocketChannel::didUpdateBufferedAmount):
(WebCore::WebSocketChannel::didFailSocketStream):
(WebCore::WebSocketChannel::didStartLoading):
(WebCore::WebSocketChannel::didReceiveData):
(WebCore::WebSocketChannel::didFinishLoading):
(WebCore::WebSocketChannel::didFail):
(WebCore::WebSocketChannel::appendToBuffer):
(WebCore::WebSocketChannel::skipBuffer):
(WebCore::WebSocketChannel::processBuffer):
(WebCore::WebSocketChannel::resumeTimerFired):
(WebCore::WebSocketChannel::startClosingHandshake):
(WebCore::WebSocketChannel::closingTimerFired):
(WebCore::WebSocketChannel::processFrame):
(WebCore::WebSocketChannel::enqueueTextFrame):
(WebCore::WebSocketChannel::enqueueRawFrame):
(WebCore::WebSocketChannel::enqueueBlobFrame):
(WebCore::WebSocketChannel::processOutgoingFrameQueue):
(WebCore::WebSocketChannel::abortOutgoingFrameQueue):
(WebCore::WebSocketChannel::sendFrame):
(WebCore::WebSocketChannel::clientHandshakeRequest const):
(WebCore::WebSocketChannel::serverHandshakeResponse const):
* Source/WebKitLegacy/WebCoreSupport/WebSocketChannel.h: Renamed from Source/WebCore/Modules/websockets/WebSocketChannel.h.
* Source/WebKitLegacy/WebKitLegacy.xcodeproj/project.pbxproj:
* Source/WebKitLegacy/mac/WebView/WebView.mm:
(-[WebView _commonInitializationWithFrameName:groupName:]):
(-[WebView initSimpleHTMLDocumentWithStyle:frame:preferences:groupName:]):

Canonical link: https://commits.webkit.org/261973@main
